examples/api/src/index.ts

```diff
@@ -142,13 +172,52 @@ async function asyncMain() {
     })
   }
 
+  const paymentProviders = [
+    new StripeCheckoutPaymentProvider({
+      id: 'stripe_checkout',
+      name: 'Stripe Checkout',
+      offSessionPayments: false,
+      secretKey: process.env.STRIPE_SECRET_KEY,
+      webhookEndpointSecret: process.env.STRIPE_CHECKOUT_WEBHOOK_SECRET,
+      incomingRequestHandler: bodyParser.raw({type: 'application/json'})
+    }),
+    new StripePaymentProvider({
+      id: 'stripe',
+      name: 'Stripe',
+      offSessionPayments: true,
+      secretKey: process.env.STRIPE_SECRET_KEY,
+      webhookEndpointSecret: process.env.STRIPE_WEBHOOK_SECRET,
+      incomingRequestHandler: bodyParser.raw({type: 'application/json'})
+    }),
+    new PayrexxPaymentProvider({
+      id: 'payrexx',
+      name: 'Payrexx',
+      offSessionPayments: false,
+      instanceName: process.env.PAYREXX_INSTANCE_NAME,
+      instanceAPISecret: process.env.PAYREXX_API_SECRET,
+      psp: [0, 15, 17, 2, 3, 36],
+      pm: [
+        'postfinance_card',
+        'postfinance_efinance',
+        // "mastercard",
+        // "visa",
+        'twint',
+        // "invoice",
+        'paypal'
+      ],
```

Comment:
Maybe this could all be stronger typed than string[] and number[] in the future? But think this leads back to server.WepublishServerOpts{... where there is already a ticket regarding all the ts-ignore needed there. So will be maybe included in a little refactoring to get more ts support. Except I misunderstood the reason for it being loose is related to the webhooks or so?

Reply:
Yes we could type it stronger and this will be a task for a small refactoring. But eventually those inputs will come from settings page in the editor and then we have better control about the content.

Reply:
Ah ok, sounds good :+1: